modules/ximgproc/misc/python/pyopencv_ximgproc.hpp

```diff
@@ -1,3 +1,5 @@
 #ifdef HAVE_OPENCV_XIMGPROC
+#include "../../include/opencv2/ximgproc/edge_drawing.hpp"
```

Comment:
Hi alalek, I've checked my `pyopencv_custom_headers.h` and it does have a line `#include "/home/guide02/Downloads/library_installing/opencv-4.5.4/modules/ximgproc/misc/python/pyopencv_ximgproc.hpp"`. 
I have just finished trying to compile and build without modifying `pyopencv_ximgproc.hpp`. This time I moved the whole opencv respository to another directory. Weirdly, the bug did not show up this time. So it might be a bug corresponding to specific environments. Now I see it is no more a problem and will close this pull request. A lot of apologize for wasting your time.